ui/src/main/java/org/jtaccuino/core/ui/JavaCellFactory.java

```diff
@@ -308,13 +316,17 @@ private void handleExecution() {
             var displayManager = shell.getExtension(DisplayExtension.class);
             var printManager = shell.getExtension(PrintExtension.class);
             shell.evalAsync(() -> {
-                displayManager.setActiveOutput(outputBox);
-                displayManager.setCurrentCellData(control.getCellData());
-                printManager.setActiveStreamResult(streamResult);
-                printManager.setCurrentCellData(control.getCellData());
-                control.getCellData().getOutputData().clear();
-                Platform.runLater(() -> streamResult.setText(""));
-                Platform.runLater(outputBox.getChildren()::clear);
+                Platform.runLater(() -> {
+                    displayManager.setActiveOutput(outputBox);
+                    displayManager.setCurrentCellData(control.getCellData());
+                    printManager.setActiveStreamResult(streamResult);
+                    printManager.setCurrentCellData(control.getCellData());
+                    control.getCellData().getOutputData().clear();
+                    Platform.runLater(() -> streamResult.setText(""));
+                    Platform.runLater(outputBox.getChildren()::clear);
```

Comment:
runLater in runLater?

```suggestion
                    streamResult.setText("");
                    outputBox.getChildren().clear();
```